packages/rke2-canal/charts/values.yaml

```diff
@@ -99,6 +99,8 @@ calico:
   felixPrometheusMetricsEnabled: true
   # Disable XDP Acceleration as we do not support it with our ubi7 base image
   felixXDPEnabled: false
+  # Configure the mask that felix selects its iptables mark bits from
+  felixIptablesMarkMask: 0xffff0000
```

Comment:
Should this be quoted? YAML natively supports hex notation, so this is equivalent to `felixIptablesMarkMask: 4294901760` and I'm not sure if the env var will handle that properly, or if it is expected to be passed in as a hex value.

Reply:
I tried to edit the daemonset manully with `kubectl edit ds rke2-canal -n kube-system`. With quotes it works. Without I get the message:
> daemonsets.apps "rke2-canal" was not valid:
* patch: Invalid value: "map[spec:map[template:map[spec:map[]]]]": cannot convert int64 to string.

I tried it with both `0xff00ff00` and `4294901760`

Also inside the yaml `FELIX_IPTABLESREFRESHINTERVAL` is quoted and set to "60".

Reply:
I mean quoted here in the values, not quoted in the template. It definitely needs to be, and already is, quoted in the template.

Reply:
Yeah, you were right. The output with helm template without the quotes was:
```yaml
- name: FELIX_IPTABLESMARKMASK
  value: "4.29490176e+09"
```

Now it looks alright with the quotes.